### DISCUSSION TITLE
3.4.6 release

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2701] [Announcements]
Minor release:
* fix crashes
* fix hang in Fit Content mode and Bookmark links

<hr /><em>This discussion was created from the release <a href='https://github.com/sumatrapdfreader/sumatrapdf/releases/tag/3.4.6rel'>3.4.6 release</a>.</em>

**Comment by ramiropistoia** [comment]
Hello, kindly fix 404 error on downloading 3.4.6 64 install.exe i tried wifi and  mobile data and doesn't work

**Comment by kjk** [comment]
I can't reproduce this i.e. I can download all versions from https://www.sumatrapdfreader.org/download-free-pdf-viewer

I did clear cloudflare cache, maybe that will fix things. Please try again.

**Reply by ramiropistoia** [reply]
It's fixed now. FYI I'm from  Argentina, maybe was the ISP or something

**Comment by GitHubRulesOK** [comment]
About box still links to old forum (still part useful)
rather than where that page back links
https://github.com/sumatrapdfreader/sumatrapdf/discussions

**Reply by kjk** [reply]
Thanks, fixed in pre-release.

**Reply by Dire-Drifter** [reply]
Hello there,

After the last cumulative update of Windows 10 21H2 the software is very glitchy - doesn't start every time, doesn't open .mobi files, hangs off as unresponding. Obviously those two don't get along!

**Reply by kjk** [reply]
For every problem you encounter, please create a separate bug report. See https://www.sumatrapdfreader.org/docs/How-to-submit-bug-reports

**Reply by Dire-Drifter** [reply]
It's a Windows problem! Since the last [update](https://support.microsoft.com/en-us/topic/june-14-2022-kb5014699-os-builds-19042-1766-19043-1766-and-19044-1766-5c81d49d-0b6e-4808-9485-1f54e5d1bb15) many other things are screwed up! The guys from Redmond again are not knowing what they are doing!
